src/components/SubTaskList/SubTaskList.jsx

```diff
@@ -17,12 +18,13 @@ export default class SubTaskList extends Component {
 
```

Comment:
This is a rather cumbersome way to assign props to a state. Could you consider the following pattern?
`constructor(props) {
    super(props);

    this.state = {
      taskRef = props.taskRef;
      subtaskList: [];
    }
}`

Then, using destructuring, you can easily access taskRef without redundant binding to a contextual `this`.

Reply:
Yea, I like the pattern you proposed, except typos like ";" and "=" inside object ;)

Reply:
True xD. 1 am isn't the best time to code